server/py/services/api/crud/model_monitoring/deployment.py

```diff
@@ -439,11 +445,18 @@ def _get_model_monitoring_controller_function(self, image: str):
         # Set the project to the job function
         function.metadata.project = self.project
 
+        # Add stream triggers
+        function = self.apply_and_create_stream_trigger(
+            function=function,
+            function_name=mm_constants.MonitoringFunctionNames.APPLICATION_CONTROLLER,
+            stream_args=config.model_endpoint_monitoring.controller_stream_args,
+        )
+
         function = self._apply_access_key_and_mount_function(
             function=function,
             function_name=mm_constants.MonitoringFunctionNames.APPLICATION_CONTROLLER,
         )
-        function.spec.max_replicas = 1
+        # function.spec.max_replicas = 1
```

Comment:
you can delete this row now
```suggestion
        # function.spec.max_replicas = 1
```